iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer.hpp

```diff
@@ -64,11 +64,13 @@ class RelativePointer : public BaseRelativePointer
     template <typename U = T>
     typename std::enable_if<!std::is_void<U>::value, const U&>::type operator*() const noexcept;
 
-    /// @brief read-only access to the underlying object
+    /// @brief read-only access to the underlying object. If the RelativePointer does not point to anything the
+    /// application might terminate.
     /// @return a pointer to the underlying object
     T* operator->() const noexcept;
 
-    /// @brief access the underlying object
+    /// @brief access the underlying object. If the RelativePointer does not point to anything the
+    /// application might terminate.
     /// @return a pointer to the underlying object
     T* get() const noexcept;
```

Comment:
Same here, breaks the const correctness and it should be:
```cxx
const T* get() const noexcept;
T* get() noexcept;
```

Reply:
Closing this thread as it's a duplicate of [this one](https://github.com/eclipse-iceoryx/iceoryx/pull/1599#discussion_r963478931).